src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractFeaturesEngine.java

```diff
@@ -131,18 +134,65 @@ public void traverse() {
                                                                                              INDEL_QUAL_THRESHOLD);
 
         final String userDefinedFunctions = ExtractFeaturesBQ.getVQSRFeatureExtractUserDefinedFunctionsString();
+        Map<String, String> labelForQuery = createQueryLabels(queryLabels);
```

Comment:
these names are really similar, maybe call this "cleanQueryLabels" or something like that?